emote/memory/memory.py

```diff
@@ -486,6 +486,30 @@ def __iter__(self):
             yield {self.data_group: data, self.size_key: data[self.size_key]}
 
 
+class MemoryLoaderList:
```

Comment:
Please see my pull request here: https://github.com/EmbarkStudios/emote/pull/181 . The current implementation risks clobbering data if multiple loaders have the same datagroup, and there is also some troubles with the size_key that each loader adds.

Reply:
Nice! I removed it from my PR then :)